sassc.c

```diff
@@ -202,6 +202,7 @@ void print_usage(char* argv0) {
     printf("   -m, --sourcemap         Emit source map.\n");
     printf("   -M, --omit-map-comment  Omits the source map url comment.\n");
     printf("   -p, --precision         Set the precision for numbers.\n");
+    printf("   -a, --sass              Force the format when using stdin.\n");
```

Comment:
Assuming the following is true:
>Even with file context, setting `sass_option_set_is_indented_syntax_src` to true will force the `*.scss` to compile as indented syntax.

the switch description seems to underestimate the effect. Would the following make it clearer:

```c
printf("   -a, --sass              Treat input file (or stdin) as indented syntax.\n");
```

?